optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/client/resources/i18n/OptaShiftUIConstants_en.properties

```diff
@@ -0,0 +1,28 @@
+ACTIONS=Actions
+ADD=Add
+CLICK_TO_START_SOLVING=Click the <i>Solve</i> button to start solving.
+DELETE=Delete
+
+EMPLOYEE=Employee
+EMPLOYEE_NAME=Employee name
```

Comment:
What would be the key for referencing the class concept "EmployeeAvailability"?
What would be the key for referencing the field concept "Employee.name"?

Ideally, the class/field -> i18n key mapping is reversable.
So FOO_BAR is either class FooBar or field Foo.bar

Reply:
The issue here is many of the constants are shared (Name, Actions, Delete, Employee, Spot) and don't really belong to any particular class. Should we have a general "class" for constants that are used in many different files? (So "Name" becomes "General.Name", class "FooBar" is "FooBar", and field "bar" of "Foo" is "Foo.bar")

Reply:
What do the wb projects do in this situation?

Reply:
Couldn't find an example in kie-wb; look like they are duplicating values.